examples/with-storybook/package.json

```diff
@@ -2,27 +2,27 @@
   "name": "with-storybook",
   "version": "0.0.0",
   "scripts": {
-    "storybook": "start-storybook -p 6006 -s public",
+    "storybook": "start-storybook -h localhost -p 6006 -s public",
```

Comment:
Storybook started to open at an absolute address now. MSW cannot be registered on an absolute address unless you a) whitelist that addresse as secure, b) load that address via HTTPS. Both options are overcomplicated for the example's purpose.